docs/docker_cheatsheet.md

```diff
@@ -17,8 +17,8 @@ docker compose build --force-rm
 Accessing `bash` in your web containers can be achieved by:
 
 ```bash
-docker exec -it safe-infrastructure_cfg-web_1 bash
-docker exec -it safe-infrastructure_txs-web_1 bash
+docker exec -it safe-infrastructure-cfg-web-1 bash
```

Comment:
This name depends on the system and can be different depending on Windows, Linux or Mac, but you could do:

```
docker compose exec -it cfg-web bash
```

I will merge if you do that

Reply:
Done